src/pages/workspace/WorkspaceNewRoomPage.tsx

```diff
@@ -41,11 +41,11 @@ function WorkspaceNewRoomPage() {
     const isFocused = useIsFocused();
     const {translate} = useLocalize();
     const {isOffline} = useNetwork();
-    const [policies] = useOnyx(ONYXKEYS.COLLECTION.POLICY);
-    const [reports] = useOnyx(ONYXKEYS.COLLECTION.REPORT);
-    const [formState] = useOnyx(ONYXKEYS.FORMS.NEW_ROOM_FORM, {initWithStoredValues: false});
-    const [session] = useOnyx(ONYXKEYS.SESSION);
-    const [activePolicyID] = useOnyx(ONYXKEYS.NVP_ACTIVE_POLICY_ID);
+    const [policies] = useOnyx(ONYXKEYS.COLLECTION.POLICY, {canBeMissing: false});
```

Comment:
@daledah could you please set all of them to `true`
```suggestion
    const [policies] = useOnyx(ONYXKEYS.COLLECTION.POLICY, {canBeMissing: true});
```

Reply:
cool, let's make it all false then